api/src/movie/movie.service.ts

```diff
@@ -10,6 +10,10 @@ export class MovieService {
     return await this.moviesDao.getMovies(userId, movieId);
   }
 
+  async getWatchedListMovies(listId?: string, userId?: string) {
```

Comment:
Same here, there should be just one `getMovies` that's flexible enough to handle all our use cases.